hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/precompile/SyntheticTxnFactory.java

```diff
@@ -508,16 +508,11 @@ public TransactionBody.Builder createTokenCreate(final TokenCreateWrapper tokenC
     public TransactionBody.Builder createAccount(
             final ByteString alias,
             final Key key,
-            @Nullable final ByteString evmAddress,
```

Comment:
Note that the parameter `evmAddress` (being removed) was annotated `@Nullable` while the parameter `alias` - here and in many other methods below) is _not_ marked `@NonNull`.  What's the coding standard in this repo?  If _not_ marked `@Nullable` then _assume_ `@NonNull`?  I ask because if that _isn't_ the coding standard here then there are a lot of null checks missing ... (and then, that may be part of the coding standard in this repo too, I don't know: it would be helpful for me to know what was the usual expectation).